images/protokube/Dockerfile

```diff
@@ -17,7 +17,7 @@ FROM k8s.gcr.io/debian-base-amd64:0.3
 # ca-certificates: Needed to talk to EC2 API
 # e2fsprogs: Needed to mount / format ext4 filesytems
 RUN apt-get update && apt-get install --yes \
-  bash ca-certificates e2fsprogs systemd \
+  bash ca-certificates e2fsprogs \
```

Comment:
Anything similar that should be done one the Bazel side?
https://github.com/kubernetes/kops/blob/afd9aa1a4de0ba413bb303c70667cd2e8d519ee6/images/BUILD.bazel#L10-L13